docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ConvertToGrayScale.cs

```diff
@@ -42,29 +43,31 @@ public static void Example()
                            .Append(mlContext.Transforms.ConvertToGrayscale("Grayscale", "ImageObject"));
 
             var transformedData = pipeline.Fit(data).Transform(data);
+
             // The transformedData IDataView contains the loaded images column, and the grayscaled column.
+            PrintColumns(transformedData);
 
-            // Preview the transformedData. 
-            var transformedDataPreview = transformedData.Preview();
-            PrintPreview(transformedDataPreview);
             // ImagePath    Name         ImageObject            Grayscale
             // tomato.bmp   tomato       System.Drawing.Bitmap  System.Drawing.Bitmap
             // banana.jpg   banana       System.Drawing.Bitmap  System.Drawing.Bitmap
             // hotdog.jpg   hotdog       System.Drawing.Bitmap  System.Drawing.Bitmap
             // tomato.jpg   tomato       System.Drawing.Bitmap  System.Drawing.Bitmap
         }
 
-        private static void PrintPreview(DataDebuggerPreview data)
+        private static void PrintColumns(IDataView transformedData)
         {
-            foreach (var colInfo in data.ColumnView)
-                Console.Write("{0,-25}", colInfo.Column.Name);
-
-            Console.WriteLine();
-            foreach (var row in data.RowView)
+            var imagePathColumn = transformedData.GetColumn<string>("ImagePath").GetEnumerator();
+            var namePathColumn = transformedData.GetColumn<string>("Name").GetEnumerator();
+            var imageObjectColumn = transformedData.GetColumn<Bitmap>("ImageObject").GetEnumerator();
+            var grayScaleColumn = transformedData.GetColumn<Bitmap>("Grayscale").GetEnumerator();
+            Console.WriteLine("{0, -25} {1, -25} {2, -25} {3, -25}", "ImagePath", "Name", "ImageObject", "Grayscale");
+            while (imagePathColumn.MoveNext() && namePathColumn.MoveNext() && imageObjectColumn.MoveNext() && grayScaleColumn.MoveNext())
             {
-                foreach (var kvPair in row.Values)
-                    Console.Write("{0,-25}", kvPair.Value);
-                Console.WriteLine();
+                Console.WriteLine("{0, -25} {1, -25} {2, -25} {3, -25}", imagePathColumn.Current, namePathColumn.Current, imageObjectColumn.Current, grayScaleColumn.Current);
```

Comment:
>Console.WriteLine("{0, -25} {1, -25} {2, -25} {3, -25}", imagePathColumn.Current, namePathColumn.Current, imageObjectColumn.Current, grayScaleColumn.Current); [](start = 16, length = 158)

I feel like we're not really doing anything interesting with this data. What does writing the bitmap to string actually do that is informative to anyone?

It seems like this is just replicating the effect of the preview dataview structure without actually inspecting why it was helpful or necessary to do that in the first place, with the end result that we now have a somewhat odd sample.